docs/CHANGELOG.md

```diff
@@ -15,6 +15,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   acceptable range and no ethtx will be submitted. As additional protection, we also now
   check the receipts of the ethtx's and if they were reverted, we mark the ethtx task as failed.
 - Improved performance of OCR jobs to reduce database load. OCR jobs now run with unlimited parallelism and are not affected by `JOB_PIPELINE_PARALLELISM`.
+- Squashed migrations into a single 1_initial migration. If you were running a version
+older than 0.9.9, you need to upgrade to 0.9.9 first before upgrading to the next
+version so that the migrations are run. The CLI also now includes the following 
+commands for database version management (example commands):
+    - `chainlink db list` lists all available database versions.
+    - `chainlink db up -d postgres://postgres:node@localhost:5432/test` migrate to latest version 
```

Comment:
open to making these env vars also, but personally I like the explicit nature of command line flags